### ISSUE TITLE
Issue with EXT:staticfilecache and "useFallbackMiddleware" option

### ISSUE THREAD

**Issue by a-r-m-i-n** [christophlehmann/httpbasicauth#8]
When "staticfilecache" is used in the system, you only get the Basic Authentication prompt for the very first request of each page. 

When you entered a valid authentication, this request will get statically cached. Any other request to this page will then **NOT** trigger the user/password prompt!

The issue here is the Fallback Middleware of staticfilecache, which prevents your middleware from being processed.

To bypass this issue, I've set the "useFallbackMiddleware" of staticfilecache to `"0"`, in the AdditionalConfiguration.php - based on the Application context.

**Comment by christophlehmann** [comment]
Good point. Also had this problem once. EXT:staticfilecache has some nice Events to come around this. Can you test the change?
